docs/synchronizations-faq.md

```diff
@@ -23,6 +23,15 @@ or you can enable synchronization for an existing Organization in _Settings_.
 
 ![Synchronization](/Resources/public/img/docs/features/Sync-20241218.png)
 
+## Accessing packages from Synchronizations across multiple integrations
+
+**Note: Organization members can only access packages from synchronizations from integrations connected to their account.**
```

Comment:
```suggestion
Organization members can only access packages from synchronizations with integrations connected to their account.
```

I would just make this part of the introduction paragraph?

Reply:
Yes, you are probably right! I thought with "Note: " in front it would be more prominent, but I guess that is not really needed.